dashboard-generator.js

```diff
@@ -122,7 +122,10 @@ function convertRequests(name, requests) {
     Object.entries(request).forEach(([key, value]) => {
       if (key === "style") {
         result += convertMapping(key, value);
-      } else {
+      } else if (key === "conditional_formats") {
+        result += convertConditionalFormats(key, value);
```

Comment:
Would using `convertMapping` work instead of creating a new function? I think they do the same thing
```suggestion
        result += convertMapping(key, value);
```

Reply:
> Would using `convertMapping` work instead of creating a new function? I think they do the same thing
Hi, thanks a lot for your response 😄 
In fact this field in datadog json is a list of mapping, but in terraform it is repeated without a list concept (strange for me too...) 
`"conditional_formats": [
                                        {
                                            "comparator": "<=",
                                            "value": 0,
                                            "palette": "white_on_green"
                                        },
                                        {
                                            "comparator": ">=",
                                            "value": 1,
                                            "palette": "white_on_yellow"
                                        },
                                        {
                                            "comparator": ">=",
                                            "value": 3,
                                            "palette": "white_on_red"
                                        }
                                    ]`
vs

            conditional_formats {
              comparator = "<="
              value      = 0
              palette    = "white_on_green"
            }
            conditional_formats {
              comparator = ">="
              value      = 1
              palette    = "white_on_yellow"
            }
            conditional_formats {
              comparator = ">="
              value      = 3
              palette    = "white_on_red"
            }

Reply:
source: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/dashboard#example-usage-create-a-new-datadog-dashboard---ordered-layout

Reply:
Ahhh sorry, I missed that subtle difference! Thanks for the great example. The solution you came up with looks good to me :) could you please edit the examples (or add a new one) in the examples directory so that we have one with conditional formats? It'd be great if you could add the .json and .tf versions. Thanks again!

Reply:
Sure, I updated the PR, just one slight note 😂:
I don't have the same json formatter as yours, so I kind of made too much changes in examples...